### DISCUSSION TITLE
Invalid copilot token: missing token: 403

### DISCUSSION THREAD

**Discussion by jiaweian1001** [community/community#114456] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

I need help, I am unable to use copilot.
I have tried:
- Sign out and sign in ;
- reinstall copilot extension
- restart vscode.

```
2024-03-27 16:05:48.389 [info] [auth] Invalid copilot token: missing token: 403 
2024-03-27 16:05:48.502 [error] [auth] Extension activation failed: "Contact Support. You are currently logged in as jiaweian1001."
```

**Comment by Akash1134** [comment, marked as answer]
Hi @jiaweian1001 ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Comment by xingbofeng** [comment]
the same problem

**Comment by xingbofeng** [comment]
<img width="749" alt="屏幕截图 2024-03-27 210557" src="https://github.com/community/community/assets/15172026/6d0dcd5e-c8eb-4f2a-96d6-caf062a92425">

but i buy copilot for 1 year

![image](https://github.com/community/community/assets/15172026/ade848bd-ea59-4319-b457-f2df981c49fb)

**Comment by CafuChino** [comment]
the same problem

**Comment by lvfen** [comment]
the same problem

**Comment by papalqi** [comment]
the same problem

**Comment by CharlieYe12** [comment]
the same problem

**Comment by xbingo** [comment]
the same problem

**Comment by scue** [comment]
The same problem.

```
2024-03-28 20:58:16.970 [info] [auth] Invalid copilot token: missing token: 403 
2024-03-28 20:58:17.087 [error] [auth] Extension activation failed: "Contact Support. You are currently logged in as scue."
```
<img width="933" alt="image" src="https://github.com/community/community/assets/2650439/6b516469-a8e8-4c8f-86cc-26cbf7a077d9">

**Comment by CafuChino** [comment]
I encountered a similar issue a few days ago. According to the response from the ticket, the cause of the issue was that Github "perceived" an excessive or automated usage behavior from my end, leading to a unilateral ban on Copilot privileges without any notification or warning, even though I may have paid for this service. However, when I requested evidence to substantiate this action, my request was denied. Instead, I was repeatedly threatened with a permanent ban on my Github account if they still "perceive" similar behavior from me.

Therefore, I would like to alert those who have encountered this problem (or have yet to encounter it) to the following points:

Github may unilaterally suspend your account at any time.
Despite subscribing and paying for a feature you need, Github can still unilaterally disable this feature when they deem necessary, without providing any notification, explanation, or compensation.
Your data on Github is not secure, as Github can permanently ban your account when they deem necessary, especially when you seek to challenge their actions, they respond with threats.
Given these circumstances, I would advise, if possible, to consider other code completion tools, backup your data as much as possible, and turn to open-source solutions to minimize reliance on Github.

After this incident, I have lost confidence in Github's service level and data security. This is because, in the same ticket, I was threatened twice with a permanent account ban, when all I wanted to know was what I had done to warrant a ban on my paid subscription feature. All they did was issue threats and direct me to read the user agreement.

**Comment by jiangyijie27** [comment]
same problem.

**Comment by QRiner** [comment]
same problem
